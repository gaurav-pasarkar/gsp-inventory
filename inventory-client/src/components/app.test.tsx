import React from 'react';
import {render} from "@testing-library/react";
import App from "./app";

it('renders correctly', () => {
  const tree = render(<App />)

  expect(tree).toMatchSnapshot();
});