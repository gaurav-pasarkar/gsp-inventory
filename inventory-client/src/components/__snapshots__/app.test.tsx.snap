// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="makeStyles-root-1"
      >
        <header
          class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary makeStyles-appBar-3 mui-fixed MuiPaper-elevation4"
        >
          <div
            class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
          >
            <button
              aria-label="open drawer"
              class="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-4 MuiIconButton-colorInherit MuiIconButton-edgeStart"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-noWrap"
            >
              Retail E-commerce
            </h6>
          </div>
        </header>
        <nav
          aria-label="mailbox folders"
          class="makeStyles-drawer-2"
        >
          <div
            class="PrivateHiddenCss-smUp-12"
          />
          <div
            class="PrivateHiddenCss-xsDown-10"
          >
            <div
              class="MuiDrawer-root MuiDrawer-docked"
            >
              <div
                class="MuiPaper-root MuiDrawer-paper makeStyles-drawerPaper-6 MuiDrawer-paperAnchorLeft MuiDrawer-paperAnchorDockedLeft MuiPaper-elevation0"
              >
                <div>
                  <div
                    class="makeStyles-toolbar-5"
                  />
                  <hr
                    class="MuiDivider-root"
                  />
                  <ul
                    class="MuiList-root MuiList-padding"
                  >
                    <div
                      aria-disabled="false"
                      class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                      role="button"
                      tabindex="0"
                    >
                      <div
                        class="MuiListItemIcon-root"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                          />
                        </svg>
                      </div>
                      <div
                        class="MuiListItemText-root"
                      >
                        <span
                          class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                        >
                          Buy Product(s)
                        </span>
                      </div>
                      <span
                        class="MuiTouchRipple-root"
                      />
                    </div>
                    <div
                      aria-disabled="false"
                      class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                      role="button"
                      tabindex="0"
                    >
                      <div
                        class="MuiListItemIcon-root"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.08-.14.12-.31.12-.48 0-.55-.45-1-1-1H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                          />
                        </svg>
                      </div>
                      <div
                        class="MuiListItemText-root"
                      >
                        <span
                          class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                        >
                          Sell Product(s)
                        </span>
                      </div>
                      <span
                        class="MuiTouchRipple-root"
                      />
                    </div>
                    <div
                      aria-disabled="false"
                      class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                      role="button"
                      tabindex="0"
                    >
                      <div
                        class="MuiListItemIcon-root"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M20 4H4v2h16V4zm1 10v-2l-1-5H4l-1 5v2h1v6h10v-6h4v6h2v-6h1zm-9 4H6v-4h6v4z"
                          />
                        </svg>
                      </div>
                      <div
                        class="MuiListItemText-root"
                      >
                        <span
                          class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                        >
                          Inventory
                        </span>
                      </div>
                      <span
                        class="MuiTouchRipple-root"
                      />
                    </div>
                  </ul>
                  <hr
                    class="MuiDivider-root"
                  />
                  <ul
                    class="MuiList-root MuiList-padding"
                  >
                    <div
                      aria-disabled="false"
                      class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                      role="button"
                      tabindex="0"
                    >
                      <div
                        class="MuiListItemIcon-root"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z"
                          />
                        </svg>
                      </div>
                      <div
                        class="MuiListItemText-root"
                      >
                        <span
                          class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                        >
                          Reports
                        </span>
                      </div>
                      <span
                        class="MuiTouchRipple-root"
                      />
                    </div>
                    <div
                      aria-disabled="false"
                      class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                      role="button"
                      tabindex="0"
                    >
                      <div
                        class="MuiListItemIcon-root"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M18 17H6v-2h12v2zm0-4H6v-2h12v2zm0-4H6V7h12v2zM3 22l1.5-1.5L6 22l1.5-1.5L9 22l1.5-1.5L12 22l1.5-1.5L15 22l1.5-1.5L18 22l1.5-1.5L21 22V2l-1.5 1.5L18 2l-1.5 1.5L15 2l-1.5 1.5L12 2l-1.5 1.5L9 2 7.5 3.5 6 2 4.5 3.5 3 2v20z"
                          />
                        </svg>
                      </div>
                      <div
                        class="MuiListItemText-root"
                      >
                        <span
                          class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                        >
                          Invoices
                        </span>
                      </div>
                      <span
                        class="MuiTouchRipple-root"
                      />
                    </div>
                  </ul>
                </div>
              </div>
            </div>
          </div>
        </nav>
        <main
          class="makeStyles-content-7"
        >
          <div
            class="makeStyles-toolbar-5"
          />
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Rhoncus dolor purus non enim praesent elementum facilisis leo vel. Risus at ultrices mi tempus imperdiet. Semper risus in hendrerit gravida rutrum quisque non tellus. Convallis convallis tellus id interdum velit laoreet id donec ultrices. Odio morbi quis commodo odio aenean sed adipiscing. Amet nisl suscipit adipiscing bibendum est ultricies integer quis. Cursus euismod quis viverra nibh cras. Metus vulputate eu scelerisque felis imperdiet proin fermentum leo. Mauris commodo quis imperdiet massa tincidunt. Cras tincidunt lobortis feugiat vivamus at augue. At augue eget arcu dictum varius duis at consectetur lorem. Velit sed ullamcorper morbi tincidunt. Lorem donec massa sapien faucibus et molestie ac.
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
          >
            Consequat mauris nunc congue nisi vitae suscipit. Fringilla est ullamcorper eget nulla facilisi etiam dignissim diam. Pulvinar elementum integer enim neque volutpat ac tincidunt. Ornare suspendisse sed nisi lacus sed viverra tellus. Purus sit amet volutpat consequat mauris. Elementum eu facilisis sed odio morbi. Euismod lacinia at quis risus sed vulputate odio. Morbi tincidunt ornare massa eget egestas purus viverra accumsan in. In hendrerit gravida rutrum quisque non tellus orci ac. Pellentesque nec nam aliquam sem et tortor. Habitant morbi tristique senectus et. Adipiscing elit duis tristique sollicitudin nibh sit. Ornare aenean euismod elementum nisi quis eleifend. Commodo viverra maecenas accumsan lacus vel facilisis. Nulla posuere sollicitudin aliquam ultrices sagittis orci a.
          </p>
        </main>
      </div>
    </div>
    <div
      aria-hidden="true"
      class="MuiDrawer-root MuiDrawer-modal"
      role="presentation"
      style="position: fixed; z-index: 1300; right: 0px; bottom: 0px; top: 0px; left: 0px; visibility: hidden;"
    >
      <div
        aria-hidden="true"
        class="MuiBackdrop-root"
        style="opacity: 0; visibility: hidden;"
      />
      <div
        data-test="sentinelStart"
        tabindex="0"
      />
      <div
        class="MuiPaper-root MuiDrawer-paper makeStyles-drawerPaper-6 MuiDrawer-paperAnchorLeft MuiPaper-elevation16"
        style="visibility: hidden; webkit-transform: translateX(-0px); transform: translateX(-0px);"
        tabindex="-1"
      >
        <div>
          <div
            class="makeStyles-toolbar-5"
          />
          <hr
            class="MuiDivider-root"
          />
          <ul
            class="MuiList-root MuiList-padding"
          >
            <div
              aria-disabled="false"
              class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemIcon-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </div>
              <div
                class="MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                >
                  Buy Product(s)
                </span>
              </div>
              <span
                class="MuiTouchRipple-root"
              />
            </div>
            <div
              aria-disabled="false"
              class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemIcon-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.08-.14.12-.31.12-.48 0-.55-.45-1-1-1H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                  />
                </svg>
              </div>
              <div
                class="MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                >
                  Sell Product(s)
                </span>
              </div>
              <span
                class="MuiTouchRipple-root"
              />
            </div>
            <div
              aria-disabled="false"
              class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemIcon-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20 4H4v2h16V4zm1 10v-2l-1-5H4l-1 5v2h1v6h10v-6h4v6h2v-6h1zm-9 4H6v-4h6v4z"
                  />
                </svg>
              </div>
              <div
                class="MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                >
                  Inventory
                </span>
              </div>
              <span
                class="MuiTouchRipple-root"
              />
            </div>
          </ul>
          <hr
            class="MuiDivider-root"
          />
          <ul
            class="MuiList-root MuiList-padding"
          >
            <div
              aria-disabled="false"
              class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemIcon-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z"
                  />
                </svg>
              </div>
              <div
                class="MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                >
                  Reports
                </span>
              </div>
              <span
                class="MuiTouchRipple-root"
              />
            </div>
            <div
              aria-disabled="false"
              class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemIcon-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M18 17H6v-2h12v2zm0-4H6v-2h12v2zm0-4H6V7h12v2zM3 22l1.5-1.5L6 22l1.5-1.5L9 22l1.5-1.5L12 22l1.5-1.5L15 22l1.5-1.5L18 22l1.5-1.5L21 22V2l-1.5 1.5L18 2l-1.5 1.5L15 2l-1.5 1.5L12 2l-1.5 1.5L9 2 7.5 3.5 6 2 4.5 3.5 3 2v20z"
                  />
                </svg>
              </div>
              <div
                class="MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                >
                  Invoices
                </span>
              </div>
              <span
                class="MuiTouchRipple-root"
              />
            </div>
          </ul>
        </div>
      </div>
      <div
        data-test="sentinelEnd"
        tabindex="0"
      />
    </div>
  </body>,
  "container": <div>
    <div
      class="makeStyles-root-1"
    >
      <header
        class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary makeStyles-appBar-3 mui-fixed MuiPaper-elevation4"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
        >
          <button
            aria-label="open drawer"
            class="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-4 MuiIconButton-colorInherit MuiIconButton-edgeStart"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
          <h6
            class="MuiTypography-root MuiTypography-h6 MuiTypography-noWrap"
          >
            Retail E-commerce
          </h6>
        </div>
      </header>
      <nav
        aria-label="mailbox folders"
        class="makeStyles-drawer-2"
      >
        <div
          class="PrivateHiddenCss-smUp-12"
        />
        <div
          class="PrivateHiddenCss-xsDown-10"
        >
          <div
            class="MuiDrawer-root MuiDrawer-docked"
          >
            <div
              class="MuiPaper-root MuiDrawer-paper makeStyles-drawerPaper-6 MuiDrawer-paperAnchorLeft MuiDrawer-paperAnchorDockedLeft MuiPaper-elevation0"
            >
              <div>
                <div
                  class="makeStyles-toolbar-5"
                />
                <hr
                  class="MuiDivider-root"
                />
                <ul
                  class="MuiList-root MuiList-padding"
                >
                  <div
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="MuiListItemIcon-root"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                        />
                      </svg>
                    </div>
                    <div
                      class="MuiListItemText-root"
                    >
                      <span
                        class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                      >
                        Buy Product(s)
                      </span>
                    </div>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </div>
                  <div
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="MuiListItemIcon-root"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.08-.14.12-.31.12-.48 0-.55-.45-1-1-1H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                        />
                      </svg>
                    </div>
                    <div
                      class="MuiListItemText-root"
                    >
                      <span
                        class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                      >
                        Sell Product(s)
                      </span>
                    </div>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </div>
                  <div
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="MuiListItemIcon-root"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20 4H4v2h16V4zm1 10v-2l-1-5H4l-1 5v2h1v6h10v-6h4v6h2v-6h1zm-9 4H6v-4h6v4z"
                        />
                      </svg>
                    </div>
                    <div
                      class="MuiListItemText-root"
                    >
                      <span
                        class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                      >
                        Inventory
                      </span>
                    </div>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </div>
                </ul>
                <hr
                  class="MuiDivider-root"
                />
                <ul
                  class="MuiList-root MuiList-padding"
                >
                  <div
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="MuiListItemIcon-root"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z"
                        />
                      </svg>
                    </div>
                    <div
                      class="MuiListItemText-root"
                    >
                      <span
                        class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                      >
                        Reports
                      </span>
                    </div>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </div>
                  <div
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="MuiListItemIcon-root"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M18 17H6v-2h12v2zm0-4H6v-2h12v2zm0-4H6V7h12v2zM3 22l1.5-1.5L6 22l1.5-1.5L9 22l1.5-1.5L12 22l1.5-1.5L15 22l1.5-1.5L18 22l1.5-1.5L21 22V2l-1.5 1.5L18 2l-1.5 1.5L15 2l-1.5 1.5L12 2l-1.5 1.5L9 2 7.5 3.5 6 2 4.5 3.5 3 2v20z"
                        />
                      </svg>
                    </div>
                    <div
                      class="MuiListItemText-root"
                    >
                      <span
                        class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                      >
                        Invoices
                      </span>
                    </div>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </div>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </nav>
      <main
        class="makeStyles-content-7"
      >
        <div
          class="makeStyles-toolbar-5"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Rhoncus dolor purus non enim praesent elementum facilisis leo vel. Risus at ultrices mi tempus imperdiet. Semper risus in hendrerit gravida rutrum quisque non tellus. Convallis convallis tellus id interdum velit laoreet id donec ultrices. Odio morbi quis commodo odio aenean sed adipiscing. Amet nisl suscipit adipiscing bibendum est ultricies integer quis. Cursus euismod quis viverra nibh cras. Metus vulputate eu scelerisque felis imperdiet proin fermentum leo. Mauris commodo quis imperdiet massa tincidunt. Cras tincidunt lobortis feugiat vivamus at augue. At augue eget arcu dictum varius duis at consectetur lorem. Velit sed ullamcorper morbi tincidunt. Lorem donec massa sapien faucibus et molestie ac.
        </p>
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
        >
          Consequat mauris nunc congue nisi vitae suscipit. Fringilla est ullamcorper eget nulla facilisi etiam dignissim diam. Pulvinar elementum integer enim neque volutpat ac tincidunt. Ornare suspendisse sed nisi lacus sed viverra tellus. Purus sit amet volutpat consequat mauris. Elementum eu facilisis sed odio morbi. Euismod lacinia at quis risus sed vulputate odio. Morbi tincidunt ornare massa eget egestas purus viverra accumsan in. In hendrerit gravida rutrum quisque non tellus orci ac. Pellentesque nec nam aliquam sem et tortor. Habitant morbi tristique senectus et. Adipiscing elit duis tristique sollicitudin nibh sit. Ornare aenean euismod elementum nisi quis eleifend. Commodo viverra maecenas accumsan lacus vel facilisis. Nulla posuere sollicitudin aliquam ultrices sagittis orci a.
        </p>
      </main>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
